dynamic_avoidance:
  VREF: 0.5 #[m/s]
  PREDICTION_TIME: 3.5 #[s]
  RESOLUTION: 0.10 #[s]
  RADIUS: 0.6 #[m]
  MAX_ANGULAR_ACCELERATION: 0.8 #[rad/ss]
  MAX_ANGULAR_VELOCITY: 1.0 #[rad/s]
  ROBOT_FRAME: "base_link"
  OBSTACLES_FRAME: "obs"
  WORLD_FRAME: "map"
  VELOCITY_TOPIC_NAME: "/velocity"
  INTERMEDIATE_PATH_TOPIC_NAME: "/intermediate_path"
